docker/README.md

```diff
@@ -0,0 +1,154 @@
+# Docker and exercism.io
+
+Optionally, one can develop (and even deploy) exercism.io using a tool
+called Docker. The primary advantage of using it is that you don't have to
+install a ton of dependencies--like a ruby version manager, ruby, postgres,
+nodejs, and a bunch of Ruby gems--on your own system. The disadvantage,
+however, is that you may need to acquaint yourself with Docker, and you
+need a 64-bit machine.
+
+For a basic overview, see [What is Docker?][] on the Docker website.
+
+**Note:** At the present time, developing the front-end via Docker
+isn't working, but this will hopefully be remedied soon.
```

Comment:
That's a good idea. How about opening an issue for it and then linking to the issue? That way whoever starts working on it can put something about that in the comment thread on the issue so we don't get people doing double the work.